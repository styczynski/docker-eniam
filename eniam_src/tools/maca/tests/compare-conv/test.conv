[tag]
tagset=morfeusz
pre=agl
post=agg

[tag]
tagset=morfeusz
pre=nagl
post=agg

[convert]
tagset_from=morfeusz
tagset_to=ikipi
override=n1:n
override=n2:n
override=n3:n
override=p1:m1
override=p2:n
override=p3:n
check=ignore
late-check=true

[tag]
tagset=ikipi
pre=bedzie
post=fut

[join]
t1_pos=praet
t2_pos=aglt
copy_attr=per
post=fin:prt

[join]
t1_pos=praet
t2_orth=by
post=conjt:ter

[join]
t1_pos=conjt
t2_pos=aglt
copy_attr=per

[tag]
pre=fin:gnd:perf
post=fut

[tag]
pre=fin:gnd:imperf
post=prs
