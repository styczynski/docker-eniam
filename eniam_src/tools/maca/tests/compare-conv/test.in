<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE cesAna SYSTEM "xcesAnaIPI.dtd">
<cesAna xmlns:xlink="http://www.w3.org/1999/xlink" version="1.0" type="lex disamb">
<chunkList>
 <chunk>
  <chunk type="s">
   <tok>
    <orth>był</orth>
    <lex><base>być</base><ctag>praet:sg:m1:imperf</ctag></lex>
    <lex><base>być</base><ctag>praet:sg:m2:imperf</ctag></lex>
    <lex><base>być</base><ctag>praet:sg:m3:imperf</ctag></lex>
   </tok>
   <ns/>
   <tok>
    <orth>em</orth>
    <lex><base>być</base><ctag>aglt:sg:pri:imperf:wok</ctag></lex>
   </tok>
  </chunk>
 </chunk>
</chunkList>
</cesAna>
