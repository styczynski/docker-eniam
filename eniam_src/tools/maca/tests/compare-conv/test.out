<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE cesAna SYSTEM "xcesAnaIPI.dtd">
<cesAna xmlns:xlink="http://www.w3.org/1999/xlink" version="1.0" type="lex disamb">
<chunkList>
 <chunk id="ch1">
  <chunk type="s">
   <tok>
    <orth>byłem</orth>
    <lex><base>być</base><ctag>fin:sg:pri:imperf:prt:m1</ctag></lex>
    <lex><base>być</base><ctag>fin:sg:pri:imperf:prt:m2</ctag></lex>
    <lex><base>być</base><ctag>fin:sg:pri:imperf:prt:m3</ctag></lex>
   </tok>
  </chunk>
 </chunk>
</chunkList>
</cesAna>
